### PR TITLE
Change the output of volume toggle

### PR DESCRIPTION
**Pull request by mr5h4nkly**
When toggling volume amixer shows some details in the minibuffer which gets quite annoying sometimes, now it simply shows "Volume on/off" and "Microphone volume on/off" instead.

### CODE REVIEW

File: desktop-environment.el
Comment (DamienCassou) [inline desktop-environment.el:360]: this result will be bad for users who have configured `desktop-environment-volume-toggle-command` to use a different tool. Can you please change the default value of the variable to echo "Volume" instead?

**Comment by DamienCassou** [comment]
Thank you very much for your PR. I wrote you some feedback.

**Comment by mr5h4nkly** [comment]
I changed the whole thing! What do you think?

File: desktop-environment.el
Comment (DamienCassou) [inline desktop-environment.el:141]: this command doesn't exist

Comment (DamienCassou) [inline desktop-environment.el:141]: instead of "group 1", did you mean "group 0"?

File: desktop-environment.el
Comment (DamienCassou) [inline desktop-environment.el:368]: I'm wondering if "Sound [on]" would be better than "Volume [on]". What do you think?

File: desktop-environment.el
Comment (DamienCassou) [inline desktop-environment.el:378]: I'm wondering if "Mic [on]" would be better than "Microphone Volume [on]". What do you think?

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Thank you very much! I think this is going in a good direction. Only minor details left and I will merge.

File: desktop-environment.el
Comment (2ndemosthenes) [reply desktop-environment.el:141]: Oh sorry, at first I was going for a different thing, forgot to change this line.

File: desktop-environment.el
Comment (2ndemosthenes) [reply desktop-environment.el:368]: Yes, That's much better

File: desktop-environment.el
Comment (mr5h4nkly) [reply desktop-environment.el]: Oh sorry, at first I was trying another solution but I forgot to remove this line.

File: desktop-environment.el
Comment (mr5h4nkly) [reply desktop-environment.el]: Yes, That's much better.

**Comment by mr5h4nkly** [comment]
I've made the mentioned changes, thanks for the review.

**Comment by mr5h4nkly** [comment]
Sorry, made the same mistake again!

**Review by DamienCassou — DISMISSED** [review]

**Review by DamienCassou — APPROVED** [review]

**Comment by DamienCassou** [comment]
Thank you very much for your work. I did some more changes before merging:
1. One function had a docstring above 80 characters. The linter complained. I suggest you run `make check` next time to verify that everything is fine.
2. You changed the name of the regexp variable without changing the functions using it. Again, a `make check` would have told you that.
3. I replaced the regexp with a simpler form generating a more optimized regexp: `(rx (or "[on]" "[off]"))` => `"\\(?:\\[o\\(?:\\(?:ff\\|n\\)]\\)\\)"`. In my previous review, I suggested you do this change yourself but I probably forgot to save my comment and it disappeared :-).

Finally, I suggest you follow these recommendations when creating PRs next time: https://github.com/magit/magit/wiki/Dedicated-pull-request-branches. This will simplify the work of the maintainer.

Thanks again for the hard work! I hope you enjoy desktop-environment.

**Comment by 2ndemosthenes** [comment]
> Finally, I suggest you follow these recommendations when creating PRs next time: [magit/magit/wiki/Dedicated-pull-request-branches](https://github.com/magit/magit/wiki/Dedicated-pull-request-branches). This will simplify the work of the maintainer.

Thank you, I will.

**Comment by mr5h4nkly** [comment]
> Finally, I suggest you follow these recommendations when creating PRs next time: [magit/magit/wiki/Dedicated-pull-request-branches](https://github.com/magit/magit/wiki/Dedicated-pull-request-branches). This will simplify the work of the maintainer.

Thank you, I will.

**Pull request merged**
